### ISSUE TITLE
Validation Failed 1: scrollId is missing

### ISSUE THREAD

**Issue by StalkAlex** [Yelp/elastalert#1810]
Here's error I see in console

```bash
WARNING:elasticsearch:GET https://*elasticsearch url*/_search/*scroll*?scroll=30s [status:400 request:0.019s]
ERROR:root:Error running query: TransportError(400, u'action_request_validation_exception', u'Validation Failed: 1: scrollId is missing;'
```

The reason of that error is redundant part in url that worked for older versions of ElasticSearch and now it should be ```https://*elasticsearch url*/_search/?scroll=30s```.

How to fix this or is it a bug?

**Comment by Qmando** [comment]
That URL is generated by the elasticsearch python library. Check that you have the correct version installed.

**Comment by Qmando** [comment]
You can use `pip freeze | grep elasticsearch` to find your current version. The major version number should match the Elasticsearch version.

It's unpinned as a dependency (https://github.com/Yelp/elastalert/blob/master/setup.py#L35) so you likely got the latest version. As the docs suggest, you should now manually install the correct version afterwards. 

`$ pip install "elasticsearch>=5,<6"`

**Comment by Qmando** [comment]
For Elasticsearch five, use `$ pip install "elasticsearch>=5,<6"`

**Comment by Qmando** [comment]
Hmm. I haven't seen this issue before. Are you using the most recent version of ElastAlert? If you are comfortable adding some debug statements in, we might be able to figure out what's happening.

Firstly, you can add ``--es_debug_trace trace.log`` and it will write the entire Elasticsearch queries into that file. You should see it make a query first with scroll=True in the query parameters, and that should return a scroll id, and then it should another query using that scroll id.

Did you manually set the `scroll_keepalive` setting in your config?

**Comment by StalkAlex** [comment]
No, I haven't specified this option. I will try to debug in couple of days.
